.codespellignore

```diff
@@ -0,0 +1,8 @@
+altho
+datas
```

Comment:
We could probably come up with a clearer word for this in various places since "data" is already plural.

Reply:
That being said, that would take a long time to fix. Ignoring it is OK, but eventually we could probably rework it.